src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -122,13 +146,8 @@ class MediaBaseViewer extends BaseViewer {
         // Try catch is needed due to weird behavior when src is removed
         try {
             if (this.mediaEl) {
-                this.mediaEl.removeEventListener('timeupdate', this.setTimeCode);
-                this.mediaEl.removeEventListener('progress', this.progressHandler);
-                this.mediaEl.removeEventListener('volumechange', this.updateVolumeIcon);
-                this.mediaEl.removeEventListener('playing', this.playingHandler);
-                this.mediaEl.removeEventListener('pause', this.pauseHandler);
-                this.mediaEl.removeEventListener('ended', this.resetPlayIcon);
-                this.mediaEl.removeEventListener('seeked', this.seekHandler);
+                this.removeEventListenersForMediaElement();
+
                 this.mediaEl.removeEventListener('loadeddata', this.loadeddataHandler);
```

Comment:
Should these also live in `removeEventListenersForMediaElement`?

Reply:
👍